modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/index/impl/TestSortedIndexStorage.java

```diff
@@ -45,7 +44,9 @@
  * Test implementation of MV sorted index storage.
  */
 public class TestSortedIndexStorage implements SortedIndexStorage {
-    private final ConcurrentNavigableMap<ByteBuffer, Set<RowId>> index;
+    private static final Object NULL = new Object();
+
+    private final ConcurrentNavigableMap<ByteBuffer, NavigableMap<RowId, Object>> index;
```

Comment:
I guess `NavigableSet` should be used instead of the map. What do you store in those values?

Reply:
Main problem in `java.util.SortedSet#first` if the set is empty it will throw an exception.